src/applications/mhv-secure-messaging/tests/components/Compose/RecipientsSelect.unit.spec.jsx

```diff
@@ -1,83 +1,122 @@
+    const val = recipientsList[0].id;
+    selectVaSelect(screen.container, val);
+    const select = screen.getByTestId('compose-recipient-select');
+
+    waitFor(() => {
+      expect(select).to.have.value(val);
+      expect(onValueChange.calledOnce).to.be.true;
+    });
     expect(onValueChange.calledWith(recipientsList[0])).to.be.true;
-    wrapper.unmount();
   });
 
-  it('displays the signature alert when a recipient with signatureRequired is selected', () => {
-    const wrapper = mount(
-      <RecipientsSelect
-        recipientsList={recipientsList}
-        onValueChange={() => {}}
-        isSignatureRequired
-      />,
-    );
-    const alert = wrapper.find('va-alert[data-testid="signature-alert"]');
+  it('displays the signature alert when a recipient with signatureRequired is selected', async () => {
+    const customProps = {
+      isSignatureRequired: true,
+    };
+    const { getByTestId } = setup({ props: customProps });
 
-    expect(alert.text()).to.contain(
-      Constants.Prompts.Compose.SIGNATURE_REQUIRED,
-    );
-    wrapper.find('VaAlert').prop('onCloseEvent')();
-    expect(alert).to.be.empty;
-    wrapper.unmount();
+    waitFor(() => {
+      const alert = getByTestId('signature-alert');
+      expect(alert).to.exist;
+      expect(alert).to.contain.text(
+        Constants.Prompts.Compose.SIGNATURE_REQUIRED,
+      );
+      expect(alert).to.have.attribute('closeable', 'true');
+    });
   });
 
   it('does not display the signature alert when a recipient without signatureRequired is selected', () => {
-    const wrapper = mount(
-      <RecipientsSelect
-        recipientsList={recipientsList}
-        onValueChange={() => {}}
-      />,
+    const screen = setup({});
+    const select = screen.getByTestId('compose-recipient-select');
+    select.value = '2';
+
+    expect(select.value).to.equal('2');
+    const alert = screen.queryByTestId('signature-alert');
+    expect(alert).to.be.null;
+  });
+
+  it('displays the CantFindYourTeam component', () => {
+    setup({});
+    const cantFindYourTeam = document.querySelector(
+      'va-additional-info[trigger="If you can\'t find your team"]',
+    );
+    expect(cantFindYourTeam).to.exist;
+
+    const EditPreferencesLink = document.querySelector(
+      'va-additional-info section a',
+    );
+    expect(EditPreferencesLink).to.exist;
+    expect(EditPreferencesLink)
+      .to.have.attribute('href')
+      .to.not.equal('');
+
+    const contactFacilityLink = document.querySelector(
+      'va-additional-info section a[href="/find-locations/"]',
     );
-    const select = wrapper.find('VaSelect');
-    select.prop('onVaSelect')({ detail: { value: '2' } });
-    const alert = wrapper.find('[data-testid="signature-alert"]');
-    expect(alert).to.be.empty;
-    wrapper.unmount();
+    expect(contactFacilityLink).to.exist;
+    expect(contactFacilityLink)
+      .to.have.attribute('href')
+      .to.not.equal('');
```

Comment:
this second expect is probably redundant since you are already including `href` in your `querySelector`. If it finds it, there is no way it can be `.equal('')`